tensorflow-mnist-input-data/Setup.hs

```diff
@@ -65,9 +66,13 @@ httpDownload url outFile = do
     let uri = fromMaybe
               (error ("Can't be: invalid URI " ++ url))
               (URI.parseURI url)
-    result <- HTTP.simpleHTTP (HTTP.defaultGETRequest_ uri)
-    HTTP.getResponseCode result >>= \case
-        (2, 0, 0) -> HTTP.getResponseBody result >>= B.writeFile outFile
+    (_, rsp)
+        <- Browser.browse $ do
+              Browser.setAllowRedirects True -- handle HTTP redirects
+              Browser.setCheckForProxy True -- detect proxy if setted
```

Comment:
Use capitalization and punctuation in the comments to match the style of the rest of the code (or delete them since the function names seem self explanatory).